internal/client/helm/chart/values.yaml

```diff
@@ -1,2 +1,13 @@
 name: my-workbench
 apps: []
+
+imagePullSecret:
+  name: image-pull-secret
```

Comment:
when deploying it many times in the same namespace, you'll erase any previous ones.